pom.xml

```diff
@@ -36,7 +36,7 @@
     <version>0.1-alpha-2-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.164.3</jenkins.version>
+        <jenkins.version>2.184</jenkins.version>
```

Comment:
Would it be possible to match the LTS in the BOM 👇

Reply:
there is no matching version for 2.184 LTS in bom. I changed to 2.190.x.

Reply:
If our bill of materials if for Jenkins 2.190.x, we should probably support Jenkins 2.190.x LTS.

See git-plugin, for example. https://github.com/jenkinsci/git-plugin/blob/f06583ba2f83d7d15054ade0bfa68ab0009f611e/pom.xml#L32

They announce that the plugin should work with Jenkins at least at the version 2.204.1 (could work with earlier versions). And if you look at their bill of materials, they are using `bom-2.204.x`.

The supported Jenkins version normally matches the bill of materials, unless there's a good reason for doing other wise.